commands/user_e2e_test.go

```diff
@@ -136,3 +140,151 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+
+func (s *MmctlE2ETestSuite) TestDeleteUsersCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Delete user", func(c client.Client) {
+		printer.Clean()
+
+		previousVal := s.th.App.Config().ServiceSettings.EnableAPIUserDeletion
+		s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = true })
+		defer func() {
+			s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = *previousVal })
+		}()
+
+		cmd := &cobra.Command{}
+		confirm := true
+		cmd.Flags().BoolVar(&confirm, "confirm",confirm,"confirm")
+
+		newUser := s.th.CreateUser()
+		err := deleteUsersCmdF(c ,cmd, []string{newUser.Email})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 1)
+		s.Len(printer.GetErrorLines(), 0)
+		
+		deletedUser := printer.GetLines()[0].(*model.User)
+		s.Require().Equal(newUser.Username, deletedUser.Username)
+
+		// expect user deleted
+		_, err = s.th.App.GetUser(newUser.Id)
+		s.Require().NotNil(err)
+		s.Require().Equal(err.Error(), "SqlUserStore.Get: Unable to find the user., user_id=store.sql_user.missing_account.const, sql: no rows in result set")
+	})
+
+	s.RunForSystemAdminAndLocal("Delete user confirm using prompt", func(c client.Client) {
+		printer.Clean()
+
+		previousVal := s.th.App.Config().ServiceSettings.EnableAPIUserDeletion
+		s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = true })
+		defer func() {
+			s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = *previousVal })
+		}()
+
+		cmd := &cobra.Command{}
+
+		// create temp file to replace stdin
```

Comment:
Nice! :smile:

Reply:
thanks, that one took me a while to work around it :laughing: